pasqal_cloud/batch.py

```diff
@@ -23,7 +23,7 @@ class Batch:
         - created_at: Timestamp of the creation of the batch.
         - updated_at: Timestamp of the last update of the batch.
         - device_type: Type of device to run the batch on.
-        - group_id: ID of the owner group of the batch.
+        - project_id: ID of the owner project of the batch.
```

Comment:
Although `group_id` is deprecated it is still an accepted argument for the `__init__` of batch. Therefore it should be documented as existing but deprecated.